frontend/public/components/utils/resource-log.tsx

```diff
@@ -94,6 +97,26 @@ const getResourceLogURL = (
   });
 };
 
+const showDebugAction = (pod: PodKind, containerName: string) => {
+  const containerStatus = pod?.status?.containerStatuses?.find((c) => c.name === containerName);
+  if (pod.status.phase === 'Succeeded') {
```

Comment:
You use optional chaining on the line above, but not here. We should be consistent. Do we know if pod is ever undefined?

Reply:
Pod comes through as a resource that is optional and is defined as `any`.  So to be safe, I added chaining to all the pod references.